src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -158,12 +153,9 @@ function FloatingActionButtonAndPopover(props) {
         if (currentRoute && ![NAVIGATORS.CENTRAL_PANE_NAVIGATOR, SCREENS.HOME].includes(currentRoute.name)) {
             return;
         }
-        // Avoid rendering the create menu for first-time users until they have dismissed the download app banner (mWeb only).
-        if (props.shouldShowDownloadAppBanner && Browser.isMobile()) {
-            return;
-        }
         Welcome.show({routes, showCreateMenu});
-    }, [props.shouldShowDownloadAppBanner, props.navigation, showCreateMenu, props.demoInfo]);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
```

Comment:
this is a revert from the original PR https://github.com/Expensify/App/pull/26182